app/renderer/components/frame/frame.js

```diff
@@ -682,15 +683,27 @@ class Frame extends React.Component {
         isSecure = false
         const parsedUrl = urlParse(this.props.location)
         ipc.send(messages.CHECK_CERT_ERROR_ACCEPTED, parsedUrl.host, this.props.tabId)
-      } else if (['warning', 'passive-mixed-content'].includes(e.securityState)) {
+      } else if (e.securityState === 'warning' ||
```

Comment:
i think we can get rid of the check for 'warning' since it was removed in https://cs.chromium.org/chromium/src/third_party/WebKit/public/platform/WebSecurityStyle.h?q=kWebSecurityStyleNeutral&dr=CSs&l=11&rcl=ec1bba9ca0f2e6d210af6f93948708a95c406ce7&dlp=chromium&dlf=src/third_party/WebKit/public/platform/WebSecurityStyle.h&dlc=1c4d759e44259650dfb2c426a7f997d2d0bc73dc&dlr=3&dlgp=third_party/WebKit/public/platform/WebSecurityStyle.h&dlgr=chromium/chromium/src&drp=chromium&drf=src/third_party/WebKit/public/platform/WebSecurityStyle.h&drc=ec1bba9ca0f2e6d210af6f93948708a95c406ce7&drr=4&drgp=third_party/WebKit/public/platform/WebSecurityStyle.h&drgr=chromium/chromium/src

Reply:
addressed in 130d7ef77722cab1c3e416d6b74efdaf59005e6a